src/Templates/src/templates/maui-blazor/MauiApp.1.csproj

```diff
@@ -31,6 +31,9 @@
         <ApplicationDisplayVersion>1.0</ApplicationDisplayVersion>
         <ApplicationVersion>1</ApplicationVersion>
 
+        <!-- To create a packaged Windows application delete the next line and update your launch settings.  See https://aka.ms/MauiTemplateUnpackaged. -->
+        <WindowsPackageType>None</WindowsPackageType>
```

Comment:
Currently the aka.ms goes to a place that does not tell you anything much besides being the docs on publishing. Is this going to go to a real place hat talks about switching?

Reply:
added the tests.